src/templates/meeting-room-schedule/meeting-room-schedule.js

```diff
@@ -0,0 +1,145 @@
+import { React, useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import dayjs from 'dayjs';
+import { IntlProvider, FormattedMessage } from 'react-intl';
+import localeDa from 'dayjs/locale/da';
+import localizedFormat from 'dayjs/plugin/localizedFormat';
+import './meeting-room-schedule.scss';
+
+/**
+ * Meeting room schedule component.
+ *
+ * @param {object} props
+ *   Props.
+ * @param {object} props.content
+ *   The meeting room schedule content.
+ * @returns {JSX.Element}
+ *   The component.
+ */
+function MeetingRoomSchedule({ content }) {
+  // Props.
+  const { backgroundColor, events, title, metaData, textAlign, backgroundImage } = content;
+  const occupiedText = content.occupiedText ? (
+    content.occupiedText
+  ) : (
+    <FormattedMessage id="occupiedText" defaultMessage="occupiedText" />
+  );
+
+  // State.
+  const [currentDate, setCurrentDate] = useState(new Date());
+  const [translations, setTranslations] = useState();
+
+  // Styling and classes.
+  const rootClasses = `template-meeting-room-schedule ${textAlign}`;
+
+  /**
+   * dayjs with localizedformat.
+   */
+  useEffect(() => {
```

Comment:
Why is this not a part of the useEffect below, are they not related?

Reply:
I think you should just gather all the useEffects that are dependent on [] into one to make it easier to read.